server/icepeak-server.nix

```diff
@@ -13,16 +13,21 @@ let
     p.requests
   ]);
   libs = with pkgs; [ ];
+  haskell = pkgs.haskellPackages;
 in
   rec {
-    icepeak-server = pkgs.haskellPackages.callPackage ./server.nix { };
-    stack = pkgs.haskellPackages.stack;
+    icepeak-server = haskell.callPackage ./server.nix { };
+    stack = pkgs.stack;
     icepeak-server-env = pkgs.buildEnv {
       name = "icepeak-server-env";
       paths = [
-        stack
+        haskell.stack
         icepeak-server
         pythonEnv
+        haskell.haskell-language-server
+        haskell.ghcide
```

Comment:
It might be possible to omit ghcide, due to it being part of HLS. Could you check?